index.html

```diff
@@ -0,0 +1,531 @@
+                  alt=""
+                />
+                <h5 class="heading__sm">Business Growing</h5>
+              </div>
+              <p class="text text--secondary">
+                the quick fox jumps over the lazy dog the quick fox jumps over
+                the lazy dog
+              </p>
+            </div>
+          </li>
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img
+                  src="./assets/users-line.svg"
+                  class="service__card-img"
+                  alt=""
+                />
+                <h5 class="heading__sm">National top 50 firms</h5>
+              </div>
+              <p class="text text--secondary">
+                the quick fox jumps over the lazy dog the quick fox jumps over
+                the lazy dog
+              </p>
+            </div>
+          </li>
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img
+                  src="./assets/stats.svg"
+                  class="service__card-img"
+                  alt=""
+                />
+                <h5 class="heading__sm">National top 50 firms</h5>
+              </div>
+              <p class="text text--secondary">
+                the quick fox jumps over the lazy dog the quick fox jumps over
+                the lazy dog
+              </p>
+            </div>
+          </li>
+          <li class="list__item">
+            <div class="service__card">
+              <div class="service__card-header">
+                <img
+                  src="./assets/award.svg"
+                  class="service__card-img"
+                  alt=""
+                />
+                <h5 class="heading__sm">Web Development</h5>
+              </div>
+              <p class="text text--secondary">
+                the quick fox jumps over the lazy dog the quick fox jumps over
+                the lazy dog
+              </p>
+            </div>
+          </li>
+        </ul>
+      </section>
+      <section class="video">
+        <header class="video__header">
+          <h2 class="heading heading--lg heading--center heading--max">
+            We are providing best business service.
+          </h2>
+          <p class="text text--center text--max text--secondary">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+        </header>
+        <div class="video__content">
+          <div class="video__img-container">
+            <img src="./assets/media bg-cover.png" alt="" class="video__img" />
+            <div class="video__icon">
+              <img src="./assets/play.svg" alt="" class="video__icon-img" />
+            </div>
+          </div>
+          <div class="video__content-bg"></div>
```

Comment:
This element is not doing anything you can remove it

Reply:
The element is acting as a dark blue background behind the video player